app/models/miq_request_task.rb

```diff
@@ -47,7 +47,7 @@ def after_request_task_create
 
   def update_and_notify_parent(upd_attr)
     upd_attr[:message] = upd_attr[:message][0, 255] if upd_attr.key?(:message)
-    update!(upd_attr)
+    update!(upd_attr.except(:miq_task_id))
```

Comment:
Shocking... if you're a method run on the queue and you accept a hash argument, that hash could be injected with a `miq_task_id` key, which totally fails in this example if you blindly try to `update` a model without that column name.